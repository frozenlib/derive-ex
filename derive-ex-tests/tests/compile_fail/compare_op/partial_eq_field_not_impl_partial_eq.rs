use derive_ex::derive_ex;

struct NotPartialEq;

#[derive_ex(PartialEq)]
struct X {
    pub x: NotPartialEq,
}

fn main() {}
