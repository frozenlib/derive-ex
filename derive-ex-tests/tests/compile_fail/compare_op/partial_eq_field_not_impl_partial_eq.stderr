error[E0277]: can't compare `NotPartialEq` with `_`
 --> tests/compile_fail/compare_op/partial_eq_field_not_impl_partial_eq.rs:7:9
  |
7 |     pub x: NotPartialEq,
  |         ^ no implementation for `NotPartialEq == _`
  |
  = help: the trait `PartialEq<_>` is not implemented for `NotPartialEq`
